src/topic_monitor.h

```diff
@@ -153,6 +155,42 @@ class TopicMonitor
     /// The topic extensibility
     OpenDDS::DCPS::Extensibility m_extensibility;
 
+#if OPENDDS_MAJOR_VERSION == 3 && OPENDDS_MINOR_VERSION >= 24
+    struct FilterTypeSupport : OpenDDS::DCPS::TypeSupportImpl {
+      FilterTypeSupport(const DynamicMetaStruct& metastruct, OpenDDS::DCPS::Extensibility exten);
```

Comment:
We probably need a style guide for this repo for things like indentation in the future.

Reply:
That's a good point.  At least for now I assume I can copy/paste code from OpenDDS without worrying about it.

Reply:
I agree, we should probably agree on some guidelines (and/or a clang-format style / configuration so we can easily enforce it), but I will say that so far I have been taking some care to make sure we're following established patterns, such as 4-spaces per indentation.

Reply:
I was also following established patterns, just different ones.
If we had a `.clang-format` file in the repo I'd be happy to use it.

Reply:
Or we could look into https://editorconfig.org/ which seems like it has good support across vim/emacs/VS/etc.